translate/src/App.tsx

```diff
@@ -5,7 +5,7 @@ import React, { useContext, useEffect, useState } from 'react';
 import './App.css';
 
 import { EntitiesList as EntitiesListContext } from './context/EntitiesList';
-import { EntityViewProvider } from '~/context/EntityView';
+import { EntityViewProvider } from '../src/context/EntityView';
```

Comment:
Dropping the ~ prefix from local paths ought not be necessary, see for example [this StackOverflow answer](https://stackoverflow.com/questions/72468249/vitest-src-folder-alias-not-resolved-in-test-files) for their configuration in Vitest.